Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -413,3 +415,278 @@ theorem locallyCompactSpace_of_hasBasis (V : ℕ → Set Y)
 end LocallyCompact
 
 end ContinuousMonoidHom
+
+end
+
+section
+
+/-!
+
+# Continuous MulEquiv
+
+This section defines the space of continuous isomorphisms between two topological groups.
+
+## Main definitions
+
+-/
+
+universe u v
+
+variable (G : Type u) [TopologicalSpace G] (H : Type v) [TopologicalSpace H]
+
+/-- Define the structure of two-sided continuous isomorphism of additive groups. -/
+structure ContinuousAddEquiv [Add G] [Add H] extends AddEquiv G H , Homeomorph G H
+
+/-- Define the structure of two-sided continuous isomorphism of groups. -/
+@[to_additive "The type of two-sided continuous isomorphism of additive groups."]
+structure ContinuousMulEquiv [Mul G] [Mul H] extends MulEquiv G H , Homeomorph G H
+
+/-- The Homeomorphism induced from a two-sided continuous isomorphism of groups. -/
+add_decl_doc ContinuousMulEquiv.toHomeomorph
+
+/-- The Homeomorphism induced from a two-sided continuous isomorphism additive groups. -/
+add_decl_doc ContinuousAddEquiv.toHomeomorph
+
+/-- Notation for a `ContinuousMulEquiv`. -/
```

Comment:
I think `@[inherit_doc]` is better than adding a docstring for notation saying "this is notation". Humans know it's notation, but they might not know what it's notation for, so we may as well just explain with the same docstring as for `ContinuousMulEquiv`. Same comment on line 453.

Reply:
Thank you for the suggestion!